lib/nbv.js

```diff
@@ -291,10 +291,13 @@ var nbv = (function() {
         return cn;
     }
 
-    function latexer(match, m1, offset, string) {
-        return katex.renderToString(m1, {
-            throwOnError: false, // we do not want to stop rendering because of bad LaTeX
-        });
+    function latexer(isDisplayMode) {
+        return  function replacer(match, m1, offset, string) {
```

Comment:
nit: there is an extra space
```suggestion
        return function replacer(match, m1, offset, string) {
```